setup.py

```diff
@@ -52,6 +75,14 @@ def run(self):
         sys.exit(errOne or errTwo)
 
 
+class DevelopCommand(setuptools.command.develop.develop):
+
+    def run(self):
+        with open(os.path.join(COALIB_DIR, COMMIT_FILE), 'w') as cf:
+            cf.write('develop\n')
```

Comment:
what is the benefit of writing 'develop' into the file.  Isn't that the same as no commit file?
The COMMIT file is mandatory because it is listed in `package_data`?

Maybe after running `setuptools.command.develop.develop.run(..)`, remove the `COMMIT` file again...?